manifests/volume.pp

```diff
@@ -137,8 +137,8 @@
 
     }
     default: {
-      fail ( 'puppet-lvm::volume: ensure parameter can only be set to ' +
-        'cleaned, absent or present' )
+      fail ( sprintf("%s%s", 'puppet-lvm::volume: ensure parameter can only ',
```

Comment:
Rake is throwing a fit but locally the only thing lint complains about is the double quotes here. If you could just change those to single I think we'll be good to go.